XenModel/FriendlyNames.resx

```diff
@@ -1277,6 +1286,15 @@
   <data name="Message.name-vcpu_qos_failed" xml:space="preserve">
     <value>VM vCPU QoS Failed</value>
   </data>
+  <data name="Message.name-vdi_cbt_metadata_inconsistent" xml:space="preserve">
+    <value>Changed block tracking disabled due to inconsistent metadata</value>
+  </data>
+  <data name="Message.name-vdi_cbt_resize_failed" xml:space="preserve">
+    <value>Changed block tracking disabled due to resizing failure</value>

```

Comment:
... due to **a virtual disk** resizing failure ...
... due to **a** snapshot failure ...